tools/pipeline-witness/Azure.Sdk.Tools.PipelineWitness/appsettings.development.json

```diff
@@ -17,8 +17,11 @@
     "QueueStorageAccountUri": "https://pipelinewitnesstest.queue.core.windows.net",
     "BlobStorageAccountUri": "https://pipelinelogstest.blob.core.windows.net",
     "CosmosAccountUri": "https://pipelinewitnesstest.documents.azure.com",
+    "GitHubWebhookSecret": "https://pipelinewitnesstest.vault.azure.net/secrets/github-webhook-secret",
```

Comment:
We probably should give this a little more specific name in KV so we know what it is for.